LIDAR_MIN_CONFIDENCE: 7
LIDAR_MAX_CONFIDENCE: 100
LIDAR_NEWOBJ_WEIGHT: 0.01
LIDAR_RX_GATE: 1.0
LIDAR_RY_GATE: 1.0
RADAR_MIN_CONFIDENCE: 22
RADAR_MAX_CONFIDENCE: 60
RADAR_CLUSTER_EPS: 0.85
RADAR_CLUSTER_MINPTS: 0
RADAR_NEWOBJ_WEIGHT: 0.1
R_GATE: 1.0
THETA_GATE: 0.08
VT_GATE: 1.0
CAM_MIN_CONFIDENCE: 7
CAM_MAX_CONFIDENCE: 30
CAM_NEWOBJ_WEIGHT: 0.01
CAM_RX_GATE: 2.0
CAM_RY_GATE: 1.0
FUSION_MIN_CONFIDENCE: 7
FUSION_MAX_CONFIDENCE: 100
LOCAL_SINGLE_WEIGHT: 0.001
FUSION_NEWOBJ_WEIGHT: 0.00001
CAMERA_TRACK_GATE: 2.5
RX_TRACK_GATE: 1.5
RY_TRACK_GATE: 1.0